src/chainstate/stacks/boot/pox_2_tests.rs

```diff
@@ -145,6 +146,31 @@ pub fn check_all_stacker_link_invariants(
     }
 }
 
+fn check_pox_print_event(event: &StacksTransactionEvent, operation_name: &str) {
```

Comment:
If I'm understanding this correctly, this will check that all the basic elements of the event-info are there but it doesn't check the extra elements that are added for some of the different types. Do those need to be tested?

Reply:
Seconded. I think that this test should verify _all_ data returned by the `print` event.